recipe/build.sh

```diff
@@ -80,12 +80,30 @@ xcb-xvmc
 "
```

Comment:
Above list is missing `xcb-render, xcb-xinput`. That's the issue with removing `.la` files

Reply:
Yeah, I noticed that. Could be avoided pretty easily with a `find` command though.